### PR TITLE
DOCS: Edits to streamline Install OpenVINO Overview Page - port to 22.1

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Porting:
https://github.com/openvinotoolkit/openvino/pull/13830/

Details:

Edit install-openvino-overview.md to list 3 options for installing OpenVINO, with Installing OpenVINO Development Tools being the recommended option for new users

### CODE REVIEW

File: docs/install_guides/installing-model-dev-tools.md [hunk 1]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.
+
+## <a name="python-developers"></a>For Python Developers
+
+If you are a Python developer, follow the steps in the <a href="#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
+
+## <a name="cpp-developers"></a>For C++ Developers
+If you are a C++ developer, you must first install OpenVINO Runtime separately to set up the C++ libraries, sample code, and dependencies for building applications with OpenVINO. These files are not included with the PyPI distribution. See the [Install OpenVINO Runtime](./installing-openvino-runtime.md) page to install OpenVINO Runtime from an archive file for your operating system.
 
 While installing OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:24]: This is not correct. We should copy what's in the PR for master.

File: docs/install_guides/installing-model-dev-tools.md [hunk 2]
```diff
@@ -92,43 +99,39 @@ mo -h
 You will see the help message for Model Optimizer if installation finished successfully.
 
 
-## For C++ Developers
+<a name="cpp-developers"></a>
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:102]:
```suggestion
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 3]
```diff
@@ -47,16 +54,16 @@ To create a virtual environment, use the following command:
...
 
    .. code-block:: sh
-   
+
       openvino_env\Scripts\activate
-     
-     
+
+
 @endsphinxdirective
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:69]: we should also copy Step 4 and 5 from the PR for master.

File: docs/install_guides/installing-openvino-overview.md [hunk 4]
```diff
@@ -14,46 +14,59 @@
...
 
-For C++ developers, you may choose one of the following installation options for OpenVINO Runtime on your specific operating system:
+The following methods are available to install OpenVINO Runtime:
 
-* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-linux.md).
-* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
-* macOS: You can install OpenVINO Runtime using an [Installer](installing-openvino-macos.md) or [Anaconda Cloud](installing-openvino-conda.md).
-* [Raspbian OS](installing-openvino-raspbian.md).
+* Linux: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on Linux](installing-openvino-linux-header.md).
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:54]: For 2022.1, we don't have the archive installation. Instead, we have Installer, APT, YUM, Conda. So this part can keep the original version.

File: docs/install_guides/installing-openvino-overview.md [hunk 5]
```diff
@@ -14,46 +14,59 @@
...
+Still unsure if you want to install OpenVINO toolkit? Check out the [OpenVINO tutorials](../tutorials.md) to run example applications directly in your web browser without installing it locally. Here are some exciting demos you can explore:
+- [Monodepth Estimation with OpenVINO](https://docs.openvino.ai/latest/notebooks/201-vision-monodepth-with-output.html)
+- [Style Transfer on ONNX Models with OpenVINO](https://docs.openvino.ai/latest/notebooks/212-onnx-style-transfer-with-output.html)
+- [OpenVINO API Tutorial](https://docs.openvino.ai/latest/notebooks/002-openvino-api-with-output.html)
 
-- [Install OpenVINO Runtime](installing-openvino-runtime.md)
+Follow these links to install OpenVINO:
 - [Install OpenVINO Development Tools](installing-model-dev-tools.md)
+- [Install OpenVINO Runtime](installing-openvino-runtime.md)
 - [Build from Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:72]: Let's keep the yocto link here.

Comment (msmykx-intel) [reply docs/install_guides/installing-openvino-overview.md:72]:
```suggestion
- [Build from Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
- [Create a Yocto Image](installing-openvino-yocto.md)
```

File: docs/install_guides/installing-openvino-overview.md (hunk 4 above)
Comment (msmykx-intel) [reply docs/install_guides/installing-openvino-overview.md:54]:
```suggestion
* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-linux.md).
```

File: docs/install_guides/installing-openvino-overview.md [hunk 6]
```diff
@@ -14,46 +14,59 @@
...
 
-For C++ developers, you may choose one of the following installation options for OpenVINO Runtime on your specific operating system:
+The following methods are available to install OpenVINO Runtime:
 
-* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-linux.md).
-* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
-* macOS: You can install OpenVINO Runtime using an [Installer](installing-openvino-macos.md) or [Anaconda Cloud](installing-openvino-conda.md).
-* [Raspbian OS](installing-openvino-raspbian.md).
+* Linux: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on Linux](installing-openvino-linux-header.md).
+* Windows: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on Windows](installing-openvino-windows-header.md).
```
Comment (msmykx-intel) [inline docs/install_guides/installing-openvino-overview.md:55]:
```suggestion
* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
```

File: docs/install_guides/installing-openvino-overview.md [hunk 7]
```diff
@@ -14,46 +14,59 @@
...
+* macOS: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on macOS](installing-openvino-macos-header.md).
```
Comment (msmykx-intel) [inline docs/install_guides/installing-openvino-overview.md:56]:
```suggestion
* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
```

File: docs/install_guides/installing-model-dev-tools.md (hunk 3 above)
Comment (msmykx-intel) [reply docs/install_guides/installing-model-dev-tools.md:69]: Done in 
https://github.com/openvinotoolkit/openvino/pull/13868/commits/e98da6c9272054071ed559846969f2697b0a358b
https://github.com/openvinotoolkit/openvino/pull/13868/commits/c0c50a5be2d92d9bceaed5f449f6d470800d768d
https://github.com/openvinotoolkit/openvino/pull/13868/commits/f79377ba7a72df532950014a58e1a41f4a76b621

File: docs/install_guides/installing-model-dev-tools.md [hunk 8]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
 * Post-Training Optimization Tool
 * Model Downloader and other Open Model Zoo tools
 
-> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. 
+The instructions on this page show how to install OpenVINO Development Tools. If you are a Python developer, it only takes a few simple steps to install the tools with PyPI. If you are developing in C++, OpenVINO Runtime must be installed separately before installing OpenVINO Development Tools.
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:11]:
```suggestion
> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 9]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
 
-## For Python Developers
+In both cases, Python 3.6 - 3.9 need be installed on your machine before starting.
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:13]:
```suggestion
<a name="python-developers"></a>
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 10]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
 
-If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:15]:
```suggestion
## For Python Developers
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 11]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
+
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:16]:
```suggestion
If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 12]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
+## <a name="python-developers"></a>For Python Developers
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:17]:
```suggestion
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 13]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
+
+If you are a Python developer, follow the steps in the <a href="#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:19]:
```suggestion
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 14]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
+
+## <a name="cpp-developers"></a>For C++ Developers
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:21]:
```suggestion

```

File: docs/install_guides/installing-model-dev-tools.md [hunk 15]
```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
...
+If you are a C++ developer, you must first install OpenVINO Runtime separately to set up the C++ libraries, sample code, and dependencies for building applications with OpenVINO. These files are not included with the PyPI distribution. See the [Install OpenVINO Runtime](./installing-openvino-runtime.md) page to install OpenVINO Runtime from an archive file for your operating system.
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:22]:
```suggestion
```

File: docs/install_guides/installing-model-dev-tools.md (hunk 1 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-model-dev-tools.md:20]: Has been resolved, see the following suggestions having been applied.

Comment (msmykx-intel) [reply docs/install_guides/installing-model-dev-tools.md:20]: Done in 
https://github.com/openvinotoolkit/openvino/pull/13868/commits/54336da4ae4cde00af1dc2c96d4f3d4e6b946821
https://github.com/openvinotoolkit/openvino/pull/13868/commits/2614c5468002bb940536fb037fceda3272df0d3e
https://github.com/openvinotoolkit/openvino/pull/13868/commits/49528ece751c985e5d998ab6e03e441b1f67e13e
https://github.com/openvinotoolkit/openvino/pull/13868/commits/915b7116abeb38d053a3bd1313fe603c4e5da6a3
https://github.com/openvinotoolkit/openvino/pull/13868/commits/6272678c61cb8e9e3fd3a2a98089b289b1e0fa14
https://github.com/openvinotoolkit/openvino/pull/13868/commits/deeb504f7c6f653c23c41113cb3cc2ac47cd7504

Comment (xu-yuan1) [reply docs/install_guides/installing-model-dev-tools.md:20]: This is even more confusing....where is "For C++ developers"? Can we apply the changes more carefully?

**Comment by xu-yuan1** [comment]
@sgolebiewski-intel @msmykx-intel The install-dev-tools.md file is totally wrong....Please compare what's in upstream/master branch. Like I mentioned to Maciej earlier, the process for installing dev tools is the same for 2022.1 and 2022.2. The differences are with installing Runtime. So the install-dev-tools.md file should be mostly the same with what's on master or releases/2022/2.

**Comment by xu-yuan1** [comment]
OK I think I understand the problem....The updates to the install-dev-tools are done in another 2022.2 PR waiting to be ported. so in this PR, let's only add the <a> tags for "For Python developers" and "For C++ developers", just to make sure that the reference in the overview page won't break so that the doc build check can pass.

File: docs/install_guides/installing-model-dev-tools.md [hunk 16]
```diff
@@ -16,13 +16,15 @@ In both cases, Python 3.6 - 3.9 need be installed on your machine before startin
...
 
 ## <a name="cpp-developers"></a>For C++ Developers
 If you are a C++ developer, you must first install OpenVINO Runtime separately to set up the C++ libraries, sample code, and dependencies for building applications with OpenVINO. These files are not included with the PyPI distribution. See the [Install OpenVINO Runtime](./installing-openvino-runtime.md) page to install OpenVINO Runtime from an archive file for your operating system.
 
 Once OpenVINO Runtime is installed, you may install OpenVINO Development Tools for access to tools like Model Optimizer, Model Downloader, Benchmark Tool, and other utilities that will help you optimize your model and develop your application. Follow the steps in the <a href="#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. In Step 4, make sure that you follow the instructions in the "C++" tab.
 
+
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:27]:
```suggestion
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 17]
```diff
@@ -16,6 +16,7 @@ In both cases, Python 3.6 - 3.9 need be installed on your machine before startin
 
 ## <a name="python-developers"></a>For Python Developers
 
+
```
Comment (msmykx-intel) [inline docs/install_guides/installing-model-dev-tools.md:19]:
```suggestion
```

**Review by xu-yuan1 — APPROVED** [review]

**Pull request merged**